CalendarApp/NUnitTestProject/BaseIntegrationTest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using NUnit.Framework;
+using NUnitTestProject.Common;
+using Microsoft.Practices.Unity;
+using GoogleCalendarService;
+using Google.Apis.Calendar.v3.Data;
+using BookingRoom.Models.GoogleCalendar;
+using BookingRoom.Models.GoogleEvent;
+using Google.Apis.Calendar.v3;
+
+namespace NUnitTestProject
+{
+    [TestFixture]
+    public class BaseIntegrationTest
```

Comment:
I think I've previously mentioned that there is no reason for this class to exist